packages/fluxible/lib/FluxibleContext.js

```diff
@@ -311,6 +322,9 @@ FluxContext.prototype.dehydrate = function dehydrate() {
     var self = this;
     var state = {
         dispatcher: (this._dispatcher && this._dispatcher.dehydrate()) || {},
+        options: {
+            optimizePromiseCallback: this._optimizePromiseCallback
+        },
```

Comment:
dehydrate the option, so that client side action execution is also controlled by this config.  And since this flag can be used by any plugin or context, it is good to be dehydrated as part of context itself (versus adding a fluxible plugin for this and dehydrating there)